### PR TITLE
Parser timetable fix after API change #123

### PR DESCRIPTION
**Pull request by davidm1766**
review for #123

### CODE REVIEW

File: APIWrapper/ResponseParser.cs [hunk 1]
```diff
@@ -27,41 +27,33 @@ public static ScheduleWeekContent ParseResponse(string myResponse)
...
+                        LessonType lessonType = ConvertLessonType(block["tu"].ToString()[0]);
+                        string teacherName = block["u"].ToString();
+                        string roomName = block["r"].ToString();
+                        string subjectShortcut = block["k"].ToString();
+                        string subjectName = block["s"].ToString();
+                        var sc = new ScheduleHourContent(int.Parse(block["b"].ToString()), false,
+                                                         lessonType, teacherName, roomName, subjectShortcut,
+                                                         subjectName, SubjectType.None, new List<string>());
+                        //-1 because API count day 1,2,3,4,5 and we store days at indexes 0,1,2,3,4
+                        weekTimetable.DaysInWeek[int.Parse(block["dw"].ToString()) - 1].BlocksInDay.Add(sc);
```
Comment (marektavac) [inline APIWrapper/ResponseParser.cs]: And what about studygroups and subjecttype?

File: Tests/BackendTest/ParserTest.cs [hunk 2]
```diff
@@ -29,47 +29,37 @@ public void ParseResponce_ParseEmptyAndCorrectBlock()
...
-                            'k': '',
-                            'g': '',
-                            'p': ''
+                            'b': '1',
+                            'tu': 'P',
+                            'u': 'Jan Janech',
+                            'r': 'RA045',
+                            's': 'Metaprogramovanie',
+                            'k': '1181S',
+                            'dw':'1'
```
Comment (marektavac) [inline Tests/BackendTest/ParserTest.cs]: This is empty block? (name of the method is ParseEmptyAndCorrectBlock)

Comment (marektavac) [reply Tests/BackendTest/ParserTest.cs]: Please fix also the filename (differ from class name)

File: APIWrapper/ResponseParser.cs (hunk 1 above)
Comment (davidm1766) [reply APIWrapper/ResponseParser.cs]: after API change there are no more subjectgroups and subjecttypes :(

**Pull request merged**